src/app/legacy/containers/PageHandlers/withContexts.jsx

```diff
@@ -60,6 +60,7 @@ const WithContexts = Component => {
             mvtExperiments={mvtExperiments}
             isNextJs={isNextJs}
             isUK={isUK}
+            counterName={atiAnalytics?.pageIdentifier ?? null}
```

Comment:
Should we consider "fixing" the counter name issue by putting the counter name onto the EventTrackingContext instead of the RequestContext? This may require a little bit of a refactor...?

Reply:
Yea I'll maybe just fix this actually. Should do it eventually, so why not now!